specs/binding-grpc-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/grpc/kafka/streams/kafka/produce/client.stream.rpc.oneway/client.rpt

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright 2021-2024 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+connect "zilla://streams/kafka0"
+    option zilla:window 8192
+    option zilla:transmission "duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                             .typeId(zilla:id("kafka"))
+                             .merged()
+                                 .capabilities("PRODUCE_ONLY")
+                                 .topic("requests")
+                                 .partition(-1, -2)
+                                 .ackMode("LEADER_ONLY")
+                                 .build()
+                             .build()}
+
+connected
+
+write zilla:data.ext ${kafka:dataEx()
+                                 .typeId(zilla:id("kafka"))
+                                 .merged()
+                                   .produce()
+                                     .deferred(0)
+                                     .partition(-1, -1)
+                                     .key("test")
+                                     .header("zilla:identity", "test")
+                                     .header("zilla:service", "example.OnewayService")
+                                     .header("zilla:method", "OnewayClientStream")
```

Comment:
I think we discussed these `zilla:service` and `zilla:method` headers not being needed by oneway mapping.
What do you think?